src/Dependencies/PooledObjects/PooledDelegates.cs

```diff
@@ -400,6 +413,19 @@ protected override Func<TResult> Bind()
                 => () => UnboundDelegate(Argument);
         }
 
+        private sealed class CreateValueCallbackWithBoundArgument<TArg, TKey, TValue>
```

Comment:
><TArg, TKey, TValue> [](http://example.com/codeflow?start=65&length=20)

Similar comments about order of type parameters/arguments.

Reply:
sure.